src/ReduxRouter.js

```diff
@@ -24,22 +24,28 @@ function getRoutesFromProps(props) {
   return props.routes || props.children;
 }
 
+function withStore(ClassComponent) {
+  return function WrappedComponent(props) {
```

Comment:
it is important to have a proper displayName to display correctly in react dev tools

e.g:

```
function withSubscription(WrappedComponent) {
  class WithSubscription extends React.Component {/* ... */}
  WithSubscription.displayName = `WithSubscription(${getDisplayName(WrappedComponent)})`;
  return WithSubscription;
}

function getDisplayName(WrappedComponent) {
  return WrappedComponent.displayName || WrappedComponent.name || 'Component';
}
```